lib/gitarro/backend.rb

```diff
@@ -259,7 +259,7 @@ def retriggered_by_comment?(pr, context)
   # this function will check if the PR contains a checkbox
   # for retrigger all the tests.
   def retriggered_by_checkbox?(pr, context)
-    return false unless pr.body.include? "[x] Re-run test \"#{context}\""
+    return false unless pr.body.match(/\[x\]\s+Re-run\s+test\s+#{context}/i)
```

Comment:
@srbarrios Do we have checkboxes other than for the changelog ignore, and for the tests?

Reply:
Nope. We don't. Right :) Only comments.

Reply:
Ah! Right.

Let's address that with a separate PR. I think there's room for a lot improvement there, not only whitespaces and case sensitivity.